### ISSUE TITLE
Test on other dataset

### ISSUE THREAD

**Issue by kaiyi98** [uvavision/RerankingTransformer#18]
Hi!
I'm testing RRT on the Oxford RobotCar dataset, using daytime images as the database and nighttime images as the query, but the retrieval results are very poor. Is this because the RRT network is not trained in this cross day/night scenario? Also, to what order of magnitude does this cross entropy loss function based loss eventually converge?
Thanks!

**Comment by fwtan** [comment]
> Hi! I'm testing RRT on the Oxford RobotCar dataset, using daytime images as the database and nighttime images as the query, but the retrieval results are very poor. Is this because the RRT network is not trained in this cross day/night scenario? Also, to what order of magnitude does this cross entropy loss function based loss eventually converge? Thanks!

Hi Kaiyi98, 

The released model was pretrained on landmark datasets. It won't work out-of-the-box for traffic/driving benmarks, e.g. Oxford RobotCar. To adapt RRT for such scenes, you may consider further re-training/fine-tuning the model. We provide an example training log file in https://github.com/uvavision/RerankingTransformer/tree/main/RRT_GLD/logs/r50_gldv1_log_retrained, which tracked the training loss of the model.

**Comment by fwtan** [comment]
Feel free to re-open the issue if you have any further questions.
